### ISSUE TITLE
gulp build fails on build-css task

### ISSUE THREAD

**Issue by jawa-the-hutt** [aurelia/skeleton-plugin#36]
Just cloned the repo and did a `gulp build` and it errored out on the build. 

```
[21:06:31] Using gulpfile D:\OneDrive\node\aurelia-form-builder\gulpfile.js
[21:06:32] Starting 'build'...
[21:06:32] Starting 'clean'...
[21:06:32] Finished 'clean' after 44 ms
[21:06:32] Starting 'build-html'...
[21:06:32] Starting 'build-css'...
[21:06:32] 'build-css' errored after 215 μs
[21:06:32] Error: Invalid glob argument: undefined
    at Gulp.src (D:\OneDrive\node\aurelia-form-builder\node_modules\vinyl-fs\lib\src\index.js:20:11)
    at Gulp.<anonymous> (D:\OneDrive\node\aurelia-form-builder\build\tasks\build.js:17:15)
    at module.exports (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\index.js:214:10)
    at Gulp.Orchestrator.start (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\index.js:134:8)
    at runNextSet (D:\OneDrive\node\aurelia-form-builder\node_modules\run-sequence\index.js:86:16)
    at Gulp.onTaskEnd (D:\OneDrive\node\aurelia-form-builder\node_modules\run-sequence\index.js:75:5)
    at emitOne (events.js:101:20)
    at Gulp.emit (events.js:188:7)
    at Gulp.Orchestrator._emitTaskDone (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\index.js:264:8)
    at D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\index.js:275:23
    at finish (D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\lib\runTask.js:21:8)
    at D:\OneDrive\node\aurelia-form-builder\node_modules\orchestrator\lib\runTask.js:52:4
    at f (D:\OneDrive\node\aurelia-form-builder\node_modules\end-of-stream\node_modules\once\once.js:17:25)
    at DestroyableTransform.onend (D:\OneDrive\node\aurelia-form-builder\node_modules\end-of-stream\index.js:31:18)
[21:06:32] 'build' errored after 66 ms
[21:06:32] Error in plugin 'run-sequence(build-css)'
```

don't know for sure, but it may be that pull request #35 is involved.

it added the following code in the build tasks, but `paths.js` doesn't define `paths.css`

```
gulp.task('build-css', function() {
  return gulp.src(paths.css)
    .pipe(gulp.dest(paths.output + 'es2015'))
    .pipe(gulp.dest(paths.output + 'commonjs'))
    .pipe(gulp.dest(paths.output + 'amd'))
    .pipe(gulp.dest(paths.output + 'system'));
});
```

adding the following to `paths.js` appears to fix the issue:

```
css: appRoot + '/*.css'
```

Wanted to double check that is the best way to do it based on the intent of that pull request.

**Comment by AshleyGrant** [comment]
This line is already in `paths.css`. See here: https://github.com/aurelia/skeleton-navigation/blob/master/skeleton-esnext/build/paths.js#L9

**Comment by jawa-the-hutt** [comment]
ok, so please forgive my ignorance here as I'm probably missing something pretty basic, but if it's fixed in the skeleton-navigation repo, but not in the skeleton-plugin repo, how does that help someone newly cloning the skeleton-plugin repo from not encountering the same issue when they go to build it for the first time?

**Comment by AshleyGrant** [comment]
You're right. My bad. I'll get this fixed tomorrow morning.

**Comment by AshleyGrant** [comment]
Fixed by #37
